README.md

```diff
@@ -259,7 +262,7 @@ yarn build
 ## ❤️ Thanks
 
 - The core implementation of the share button code was copied from the [@liady](https://github.com/liady) extension with some modifications.
-- Thanks to the [Awesome ChatGPT Prompts](https://github.com/f/awesome-chatgpt-prompts) repository for inspiring the custom command function for this application.
+- Thank you to  [Awesome ChatGPT Prompts](https://github.com/f/awesome-chatgpt-prompts) repository for inspiring the custom command function for this application.
```

Comment:
Remove the `to` after the `Thank you`. It should read `Thank you [Awesome ChatGPT Prompts] repository...`

Reply:
Such a nit that I feel weird even chiming in, but I disagree.

This isn't a message directed at a person who identifies as the  GitHub repository... it's a general acknowledgement of the repository.